### DISCUSSION TITLE
Libre 3 working with companion mode?

### DISCUSSION THREAD

**Discussion by blaqone** [NightscoutFoundation/xDrip#2476] [Q&A]
Short question: Is the companion mode able to grab data from FL3?

**Comment by Navid200** [comment]
The companion mode gets data from another app running on the same phone.

**Reply by blaqone** [reply]
Yes i know - I’m using it with the g7 and I was wondering if it can pick up data from the FL3 app‘s message center notifications?

**Comment by jamorham** [comment]
No it isn't able to.

**Reply by blaqone** [reply]
I assume it does not work because the data is only visible within the app?

**Comment by Lovejoy74** [comment]
Dexcom one doesn't seem to work as it doesn't have share functionality
